en/user_guide/workflow.rst

```diff
@@ -0,0 +1,53 @@
+Workflow
+========
+
+Open orchestrated allows to have a publication workflow.
+It is a series of steps to achieve for publication of a node or a content before it can be displayed on Front Office.
+
+Status
+------
+
+Each node or content has a status per version, changing it allows to pass the publication workflow steps.
```

Comment:
Maybe something like "Each version of a node or a content has a status" is more readable